ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py

```diff
@@ -0,0 +1,25 @@
+from ADR.Components.Aerodynamic_components.Aerodynamic_surface import Aerodynamic_surface
+import numpy as npt
+
+
+def test_Aerodinamics_surface():
+    airfoil_Parameters = {"airfoil_clmax": 1.2, "airfoil1_name": "s1223", "airfoil2_name": "s1223",
+                          "airfoil3_name": "s1223", "span1": 2, "span2": 1.5, "chord1": 0.40, "chord2": 0.30,
+                          "chord3": 0.35, "twist1": 0, "twist2": 0, "twist3": 0, "incidence": 0}
+    aerodynamics_surfase = Aerodynamic_surface(airfoil_Parameters)
+    npt.assert_almost_equal(
+        aerodynamics_surfase.update_alpha(20), 20, decimal=0)
```

Comment:
This will throw an error as the update_alpha() method returns nothing.

With a clmax of 1.2 (from the parameters) the airfoil will not reach 20 degrees of angle of attack. See for yourself.
This is a sign (coming from the test, nice) that we should refactor the method itself, as it has problems.
This is also a sign that we should test this update_alpha function for more than one case (negative alpha, positive alpha, large positive and negative alpha, 0, etc).